pkg/ccl/changefeedccl/helpers_test.go

```diff
@@ -910,6 +910,75 @@ func expectNotice(
 	require.Equal(t, expected, actual)
 }
 
+// expectNoticeGetJobID is used for checking duplicate changefeed notices when creating changefeeds.
+// If multiple notices returned, it checks all to see that the expected notice is present
+// exactly once.
+func expectNoticeGetJobID(
```

Comment:
nit: change the name to make it more explicit that this is some super specific helper to do with dupe checking.